packages/mui-material/src/Slider/Slider.d.ts

```diff
@@ -296,6 +301,24 @@ export declare const SliderTrack: React.FC<SliderTrackProps>;
 export declare const SliderThumb: React.FC<SliderThumbProps>;
 export declare const SliderValueLabel: React.FC<SliderValueLabelProps>;
 
+interface SliderType {
```

Comment:
Let's export this type, so it can be used to make `styled` work (`const StyledSlider = styled(Slider)`` as SliderType`)
Also, a comment with a few words of explanation on what's going on here would be helpful for anyone looking at the code in the future